src/common/modules/tags/tags-panel/TagsPanel.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div>
-        <p-panel-top :use-total-count="true" :total-count="tags.length">
+        <p-panel-top :use-total-count="true" :total-count="Object.keys(tags).length">
```

Comment:
```suggestion
        <p-panel-top :use-total-count="true" :total-count="items.length">
```

이미 내부적으로 `Object.keys(state.tags)` 를 수행한 결과로 `items` state 가 존재하므로, 중복으로 할 것 없이,  `items`를 쓰면 좋을 것 같습니다.